src/app/components/Navbar/NavbarLink.tsx

```diff
@@ -14,7 +14,7 @@ function NavbarLink({ children, end = false, href }: Props) {
       to={href}
       className={({ isActive }) =>
         classNames(
-          "block font-semibold px-1 text-md",
+          "block font-semibold hover:text-slate-900 transition-all  px-1 text-md",
```

Comment:
Can you please use the colors defined in #2259? 

> Let's use gray600 for light theme and gray300 for dark for hover states, so it's not the same as active. I already updated Master file with this

Reply:
sure I'll update